src/main/java/com/xceptance/xlt/engine/util/DefaultWebDriverFactory.java

```diff
@@ -72,8 +69,7 @@
      xlt,
      firefox_clientperformance,
      chrome_clientperformance,
-     edge,
-     opera
+     edge
```

Comment:
Do we really want to drop support for Opera entirely? Or do want to add support for Chromium-based browsers such as recent Opera and Edge? Last but not least, is explicit support for Edge still necessary?

Reply:
Dedicated driver server for recent Opera can be downloaded [here](https://github.com/operasoftware/operachromiumdriver/releases)

Reply:
I'm okay for the Edge case ;) However, there might be people who want to run tests with Chrome/Chromium and Opera simultaneously; that's not possible any longer unless they provide their own WebDriver instance.

What I'm trying to say is: if Opera can be used by simply using the ChromiumDriver (not ChromeDriver) and the opera driver server, we might why not supporting that. OperaDriver might have been removed from Selenium because is became obsolete due to ChromiumDriver.